code/datums/uplink_item.dm

```diff
@@ -850,19 +833,19 @@ var/list/uplink_items = list()
 	name = "Pickpocket Gloves"
 	desc = "Gloves that allow you to quickly and stealthily remove items from people and puts them straight into your hands."
 	item = /obj/item/clothing/gloves/pickpocket
-	cost = 4
+	cost = 2
 
 /datum/uplink_item/stealthy_tools/voice_changer
 	name = "Voice Changer"
 	item = /obj/item/clothing/mask/gas/voice
 	desc = "A conspicuous gas mask that mimics the voice named on your identification card. It can be toggled on and off."
-	cost = 3
+	cost = 2
 
 /datum/uplink_item/stealthy_tools/chameleon_proj
 	name = "Chameleon Projector"
 	desc = "Projects an image across a user, disguising them as an object scanned with it, as long as they don't move the projector from their hand. Disguised users move slowly, and projectiles pass over them."
 	item = /obj/item/device/chameleon
-	cost = 7
+	cost = 6
```

Comment:
Ngl, chameleon projector is legit amazing but I never see it in use.
The amount of shit you can pull off with this is great so this seeing a little more play is fine.